alembic/versions/b206c80f7640_add_recording_inputevent_screenshot_.py

```diff
@@ -10,75 +10,99 @@
 
 
 # revision identifiers, used by Alembic.
-revision = 'b206c80f7640'
+revision = "b206c80f7640"
 down_revision = None
 branch_labels = None
 depends_on = None
 
 
 def upgrade() -> None:
     # ### commands auto generated by Alembic - please adjust! ###
-    op.create_table('recording',
-    sa.Column('id', sa.Integer(), nullable=False),
-    sa.Column('timestamp', sa.Integer(), nullable=True),
-    sa.Column('monitor_width', sa.Integer(), nullable=True),
-    sa.Column('monitor_height', sa.Integer(), nullable=True),
-    sa.Column('double_click_interval_seconds', sa.Numeric(asdecimal=False), nullable=True),
-    sa.Column('double_click_distance_pixels', sa.Numeric(asdecimal=False), nullable=True),
-    sa.Column('platform', sa.String(), nullable=True),
-    sa.PrimaryKeyConstraint('id', name=op.f('pk_recording'))
```

Comment:
Surprising that alembic autogenerates incorrectly formatted migration scripts. Please feel free to create a new issue on their github and link back to here: https://github.com/alembic/alembic/issues 🙏